facade/mock/pubKeyConverterMock.go

```diff
@@ -0,0 +1,29 @@
+package mock
```

Comment:
Do you think you can remove this mock and always use the real converter for all our tests?